daemon/checkpoint.go

```diff
@@ -76,6 +84,23 @@ func (daemon *Daemon) CheckpointCreate(name string, config types.CheckpointCreat
 		return fmt.Errorf("cannot checkpoint container %s: %s", name, err)
 	}
 
+	checkpointInfo := checkPoint{
```

Comment:
Could you add a space before the curly bracket, please?

Reply:
If I add a space, `gofmt` will remove it.

Reply:
And I prefer to add a space, it looks more pretty. But we need to upgrade `gofmt`.